components/SplitPane/desktop/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import org.jetbrains.compose.compose
+
+plugins {
+    kotlin("multiplatform")
+    id("org.jetbrains.compose")
+}
+
+kotlin {
+    jvm {}
+    sourceSets {
+        named("jvmMain") {
+            dependencies {
+                implementation(compose.desktop.currentOs)
+                implementation(project(":SplitPane:common"))
+            }
+        }
+    }
+}
+
+compose.desktop {
+    application {
+        mainClass = "org.jetbrains.compose.splitpanedemo.MainKt"

```

Comment:
"org.jetbrains.compose.splitpane.demo.MainKt" I guess

Reply:
Oops. I thought I was fix that already.